sorbonne-student-law-review.csl

```diff
@@ -0,0 +1,575 @@
+    <updated>2020-08-04T12:00:00+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="fr-FR">
+    <terms>
+      <term name="editor" form="short">dir.</term>
+      <term name="retrieved">consulté le</term>
+      <term name="presented at">thèse</term>
+    </terms>
+  </locale>
+  <locale xml:lang="en">
+    <terms>
+      <term name="retrieved">retrieved on</term>
+      <term name="presented at">doctoral thesis</term>
+      <term name="open-quote">“</term>
+      <term name="close-quote">”</term>
+      <term name="open-inner-quote">'</term>
+      <term name="close-inner-quote">'</term>
+      <term name="page-range-delimiter">-</term>
+    </terms>
+  </locale>
+  <macro name="author">
+    <choose>
+      <if type="legal_case" match="any">
+        <text variable="authority"/>
+      </if>
+      <else-if match="any" variable="author">
+        <names variable="author"/>
+      </else-if>
+      <else-if match="any" variable="editor collection-editor">
+        <choose>
+          <!-- for chapter and paper-conference, no author implies the editor is also the author. Name should remain in editor position-->
+          <if type="chapter paper-conference" match="none">
+            <names variable="editor collection-editor">
+              <label form="short" prefix=" (" suffix=")"/>
+            </names>
+          </if>
+        </choose>
+      </else-if>
+      <else>
+        <choose>
+          <if match="none" type="entry-encyclopedia entry-dictionary bill legislation chapter paper-conference report">
+            <text term="anonymous"/>
+          </if>
+        </choose>
+      </else>
+    </choose>
+  </macro>
+  <macro name="editor">
+    <choose>
+      <if match="any" type="chapter paper-conference entry-encyclopedia entry-dictionary">
+        <group delimiter="&#160;">
+          <text term="in" font-style="italic"/>
+          <names variable="editor collection-editor">
+            <label form="short" prefix=" (" suffix=")"/>
+          </names>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="translator">
+    <group delimiter="&#160;">
+      <text term="translator" form="short"/>
+      <names variable="translator"/>
+    </group>
+  </macro>
+  <macro name="author-biblio">
+    <choose>
+      <if type="legal_case" match="any">
+        <text variable="authority"/>
+      </if>
+      <else-if match="any" variable="author">
+        <names variable="author">
+          <name name-as-sort-order="all" sort-separator="&#160;">
+            <name-part name="family" text-case="capitalize-first" font-variant="small-caps"/>
+            <name-part name="given" text-case="capitalize-first" font-variant="normal" prefix="(" suffix=")"/>
```

Comment:
@p-heckler, you're encapsulating given names with parentheses here, which is probably not what you want.